app/controllers/sessions_controller.rb

```diff
@@ -6,7 +6,12 @@ def create
     UserFacade.create_user(user_params)
     
     session[:user_id] = user.id
-    redirect_to dashboard_path
+    redirect_to dashboard_path, notice: "Logged in as #{user.name}"
```

Comment:
the dashboard already has a `welcome #{user.name}` feature when logging in. Do you think we need the notice as well?